Source/KSCrash/Source/KSCrash/Recording/Tools/BSG_KSSafeCollections.m

```diff
@@ -62,10 +58,6 @@ - (void)bsg_ksc_setObjectIfNotNil:(id)object forKey:(id)key {
     }
 }
 
-- (void)bsg_ksc_safeSetObject:(id)object forKey:(id)key {
```

Comment:
This file and its tests should be able to be removed altogether. The only remaining method in use is `setObjectIfNotNil`, which can be replaced with `BSGDictInsertIfNotNil()`.